lib/hooks.js

```diff
@@ -12,67 +10,6 @@
 
 const AsyncSeriesWaterfallHook = require('tapable').AsyncSeriesWaterfallHook;
 
-// The following typedef holds the API definition for all available hooks
```

Comment:
The idea was to explain which kind of hooks can be used.  

Could we at least also move this comment to the typings file?

Reply:
Ah. Since that also applies outside of the TS world probably shouldn't go in the typings file. I'll add some documentation back in.

Reply:
Added back in via 0d0160f